interview-questions.md

```diff
@@ -7,44 +7,68 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: The student model will be the Foreign Key. It belongs_to :cohort
 
+  Researched answer: so the foreign key relates the two databases. If you forget to create the foreign key you can execute an an alter script to add the foreign key after table creation. 
```

Comment:
Haven't heard it called an alter script before.  You would need to run a migration to correct it.